File: CS50x/6. Python/15.py
'''
> Date Created: December 25, 2023
> Author: Ishaan Rastogi
> Purpose: To store scores
> Operating System: This is only for Windows OS, it may or may not work on other OS
> Program Status: 100% Working 
'''

'''

Topic- List, Tuple, Range

'''

# If the scores are to inputed manually in the code: scores = [72, 73, 33]

#If the scores are to be user input
scores=[] #empty list which is yet to be filled

for i in range(3):
    int = int(input('Score: '))
    scores.append(int) #append function- add at the end

    '''
    Another way- scores += [int]
    This is concatenation & int is put in rectangular brackets so that it is treated as a string
    '''

average = sum(scores) / len(scores)
print(f"Average: {average}") #formatted string, LEARN THIS

#Similarly, functions like upper and lower to change the case of the string can be used like this

'''
Ctrl + Shift + `
Terminal Code Loading...

python 15.py
'''

File: College/15.c
/*

> Date Created: October 15, 2023
> Author: Ishaan Rastogi
> Purpose: WAP in C to find gross salary of employee if DA is 40% of basic salary & HRA is 20% of basic salary. (User Input: Basic Salary)
> Operating System: This is only for Windows OS, it may or may not work on other OS
> Program Status: 100% Working

*/

#include <stdio.h>
int main() {
    float basicSalary, grossSalary, da, hra;

    // User input of basic salary
    printf("Enter the basic salary: ");
    scanf("%f", &basicSalary);

    // Calculating DA
    da = 0.4 * basicSalary;

    // Calculating HRA
    hra = 0.2 * basicSalary;

    // Calculating gross salary
    grossSalary = basicSalary + da + hra;

    // Display the gross salary
    printf("Gross Salary = %f\n", grossSalary);
    return 0;
}

/* 

Ctrl + Shift + `

Terminal Code Loading...

gcc 15.c
./a.exe
 
*/